### PR TITLE
Referrals bug fixes

### PR DESCRIPTION
**Pull request by dylanlott**

### CODE REVIEW

File: lib/queries/generate-referral-credits.js [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 const async = require('async');
 const constants = require('../constants');
-
-const REFERRAL_CREDIT_TYPE = constants.CREDIT_TYPE.AUTO;
+const REFERRAL_CREDIT_TYPE = constants.CREDIT_TYPES.AUTO;
```
Comment (bryanchriswhite) [inline lib/queries/generate-referral-credits.js:5]: @dylanlott @barbaraliau TODO: figure out what gives with not being able to destructure constants from service-storage-model module:
```javascript
const { CREDIT_TYPES, PROMO_CODE, STRIPE, PROMO_EXPIRES, PROMO_AMOUNT } = require('storj-service-storage-models/lib/constants');
//-- https://github.com/Storj/billing/blob/master/lib/server/routes/credits.js#L10
```

File: lib/constants.js [hunk 2]
```diff
@@ -4,25 +4,56 @@
...
+  },
   CREDIT_TYPES: {
     AUTO: 'automatic',
     MANUAL: 'manual'
   },
   DEBIT_TYPES: {
-    AUDIT: 'audit',
-    TRANSFER: 'transfer',
+    STORAGE: 'storage',
+    BANDWIDTH: 'bandwidth',
```
Comment (bryanchriswhite) [inline lib/constants.js:54]: Any constants not being used should be cleaned out after switching to referencing service-storage-models' constants.

**Review by bryanchriswhite — APPROVED** [review]

**Comment by phutchins** [comment]
@bryanchriswhite and I have reviewed and looks good. Merging.

File: lib/queries/generate-referral-credits.js (hunk 1 above)
Comment (barbaraliau) [reply lib/queries/generate-referral-credits.js:5]: This should work...when @dylanlott and I tested this route it generated the credits correctly.

**Pull request merged**